### PR TITLE
fix: rename lable `restarts` to `auto restarts`

### PR DESCRIPTION
**Pull request by keita-determined**
## Description

[DET-7513]

### Before
image

### After

Screen Shot 2022-07-13 at 10 50 26 AM

## Test Plan

Verify `Restarts` labels in `Trials` tab and `ExperimentDetails.tsx` are renamed to `Auto Restarts`. 

## Commentary (optional)

I also changed variable names from `restarts` to `autoRestarts` or similar name.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-7513]: https://determinedai.atlassian.net/browse/DET-7513?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hkang1 — COMMENTED** [review]
The rename mostly looks good. Take a look at the note about `services/api-ts-sdk/api.ts`, it's an auto generated file. A simple undo of those changes + slight translating decoder change should do it.

File: webui/react/src/services/decoder.ts [hunk 1]
```diff
@@ -503,14 +503,14 @@ export const decodeCheckpoint = (data: Sdk.V1Checkpoint): types.CoreApiGenericCh
 
 export const decodeV1TrialToTrialItem = (data: Sdk.Trialv1Trial): types.TrialItem => {
   return {
+    autoRestarts: data.autoRestarts,
```
Comment (trentwatt) [inline webui/react/src/services/decoder.ts:506]:
```suggestion
    autoRestarts: data.restarts,
```

Comment (keita-determined) [reply webui/react/src/services/decoder.ts]: Thanks. I fixed it

File: webui/react/src/pages/ExperimentDetails/ExperimentTrials.settings.ts [hunk 2]
```diff
@@ -24,18 +24,18 @@ export const DEFAULT_COLUMNS: TrialColumnName[] = [
...
   'startTime',
   'duration',
-  'restarts',
+  'autoRestarts',
   'checkpoint',
 ];
 
 export const DEFAULT_COLUMN_WIDTHS: Record<TrialColumnName, number> = {
   action: 46,
+  autoRestarts: 117,
```
Comment (hkang1) [inline webui/react/src/pages/ExperimentDetails/ExperimentTrials.settings.ts:33]: Just a heads up - I think this portion seems to be ok and not impacted, but some id renames that cause local storage keys to no longer map to the new renamed key, can cause the UI to not load properly.

**Review by hkang1 — APPROVED** [review]
LGTM!

File: webui/react/src/pages/ExperimentDetails/ExperimentTrials.settings.ts (hunk 2 above)
Comment (keita-determined) [reply webui/react/src/pages/ExperimentDetails/ExperimentTrials.settings.ts:33]: I'll keep it in mind. Thanks for letting me know!

**Pull request merged**